src/huntsman/pocs/dome/musca.py

```diff
@@ -150,8 +133,8 @@ def open(self):
                 'Dome shutter battery voltage too low to open: {!r}', v)
             return False
 
-        self._write_musca(Protocol.OPEN_DOME, 'Opening dome shutter')
-        time.sleep(HuntsmanDome.MOVE_TIMEOUT)
+        self._write_musca(Protocol.OPEN_DOME, 'Opening dome shutter.')
+        time.sleep(self._move_timeout)
```

Comment:
Is this just to wait until it's open?  Seems better to loop and check the `v == Protocol.OPEN` below and set a `max_wait` timer.